readthedocs/projects/tasks/builds.py

```diff
@@ -626,6 +628,22 @@ def get_valid_artifact_types(self):
                         },
                     )
 
+                if artifact_format_files == 1:
```

Comment:
We already check if number of files is more than one or 0, we don't need this conditional.